ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/EntityTypeToggleGroup.tsx

```diff
@@ -1,7 +1,55 @@
 import React from 'react';
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
 
-function EntityTypeToggleGroup() {
-    return <>Entity type toggle group</>;
+import useURLStringUnion from 'hooks/useURLStringUnion';
+import { entityTabValues, EntityTab } from './types';
+
+type EntityTabToggleGroupProps = {
+    entityContext?: EntityTab;
+    cveCount?: number;
+    imageCount?: number;
+    deploymentCount?: number;
+};
+
+function EntityTabToggleGroup({
+    entityContext,
+    cveCount = 0,
+    imageCount = 0,
+    deploymentCount = 0,
```

Comment:
opted to pass these in as props so that we can reuse the component throughout the feature. i still need to wire up the queries in the places its called though